File: code/problems/max/maximum.py
def maximum(first,second,third):
    if second>=first and second>=third:
        return second
    if third>=first and third>=second:
        return third
    return first
print(maximum(1,2,3));
print(maximum(3,4,2));
print(maximum(5,4,3));
print(maximum(2,2,1));

def max_of_array(arr):
    maximum=arr[0]
    for i in arr:
        if i>maximum:
            maximum=i
    return maximum

arr=[1,2,5,4,3]
print(max_of_array(arr))

File: code/problems/max/maximum.js
function maximum(first,second,third)
{
	if (first>=second && first>=third)
	{
		return first;
	}
	if (second>=first && second>=third)
	{
		return second;
	}
	return third;
}

console.log(maximum(1,2,3),maximum(4,3,2),maximum(2,4,3),maximum(2,2,1));

let arr=[1,2,3,4,5];
let maxOfArray=(arr)=>{
	let maximum=arr[0];
	for(let element of arr)
	{
		if (element>maximum)
		{
			maximum=element;
		}
	}
	return maximum;
}

console.log(maxOfArray(arr));
